plugins/catalog-backend-module-github/src/lib/github.ts

```diff
@@ -144,7 +157,11 @@ export async function getOrganizationUsers(
     query,
     r => r.organization?.membersWithRole,
     mapper,
-    { org, email: tokenType === 'token' },
+    {
+      org,
+      email: tokenType === 'token',
+      verifiedEmail: organizationVerifiedDomainEmails,
```

Comment:
Is there no way of checking whether the token has the right abilities?

What happens if we query for organizationVerifiedDomainEmails even if the token isn't capable - will the query be rejected overall, or will we just get null / an empty array there?

What I'm looking for is of course some way of simplifying this to not require any user interaction.

Reply:
If the token doesn't have access, the value will be an empty array in the response.

Reply:
Agreed, included in the end implementation